include/wx/strvararg.h

```diff
@@ -207,9 +207,17 @@ class WXDLLIMPEXP_BASE wxFormatString
         Arg_Unknown     = 0x8000     // unrecognized specifier (likely error)
     };
 
+#if wxDEBUG_LEVEL
     // Validate all format string parameters at once: the vector contains the
     // format specifiers corresponding to the actually given arguments.
     void Validate(const std::vector<int>& argTypes) const;
+#else
+    // Also provide a trivial implementation of Validate() doing nothing in
```

Comment:
Sorry for the nitpick, but this "Also" doesn't make sense any more after moving the comment. Let's just say
```suggestion
    // Provide a trivial implementation of Validate() doing nothing in
```

Reply:
Happy to, and actually stopped a minute to look at the entire #if...#else...#endif block and update both comments from the perspective as reading it as one chunk of code rather than two discrete definitions separated by many lines.